activestorage/test/service/mirror_service_test.rb

```diff
@@ -61,4 +61,8 @@ class ActiveStorage::Service::MirrorServiceTest < ActiveSupport::TestCase
         @service.url(@key, expires_in: 2.minutes, disposition: :inline, filename: filename, content_type: "text/plain")
     end
   end
+
+  test "path for file in primary service" do
+    assert_equal @service.primary.path_for(@key), @service.url_for(@key)
```

Comment:
Sorry, I used `url_for` in my example when I meant `path_for`. 🤦‍♂️

```suggestion
    assert_equal @service.primary.path_for(@key), @service.path_for(@key)
```

Reply:
Sorry. My bad. I should have checked it.